src/test/java/com/epam/novostroinyi/ui/jbehave/definition/LaunchStepDefinition.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.novostroinyi.ui.jbehave.definition;
+
+import static com.epam.novostroinyi.core.assertion.Assert.assertHasSize;
+import static com.epam.novostroinyi.core.util.WebDriverUtils.openUrl;
+
+import com.epam.novostroinyi.core.config.Property;
+import com.epam.novostroinyi.core.ui.element.UiElementsCollection;
+import com.epam.novostroinyi.core.util.WebDriverUtils;
+import com.epam.novostroinyi.step.LaunchesSteps;
+import com.epam.novostroinyi.step.LoginPageSteps;
+import com.epam.novostroinyi.step.TestItemsSteps;
+import lombok.SneakyThrows;
+import org.jbehave.core.annotations.AfterScenario;
+import org.jbehave.core.annotations.BeforeScenario;
+import org.jbehave.core.annotations.Given;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.annotations.When;
+
+public class LaunchStepDefinition {
+
+  private LaunchesSteps launchesSteps;
+
+  @BeforeScenario
```

Comment:
it makes sense to move beforeScenario and AfterScenario to some baseStepDefinition or smth like that since it will be copied and pasted in each stepDEefinitions classes. Or you can create test runner specifically to BBD and keep them there